packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -213,7 +214,20 @@ module.exports = {
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
-    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+    // Trigger some typechecking if a file matches with an @ flow comment
+    new FlowTypecheckPlugin({
+      flowconfig: [
+        '[ignore]',
+        '<PROJECT_ROOT>/node_modules/fbjs/.*',
+        '',
+        '[libs]',
+        './flow-typed'
```

Comment:
So, yea, unfortunately, for now, we don't have any way to trigger on `App.test.js` alone. You have to add it to`App.js`. However, once you change `App.js`, a rebuild will be retriggered including the test. In this case, it's random, sometimes flow-typed is correctly seen, sometimes not.